compiler/modules/print.wasm/tests/printBoolean.bal

```diff
@@ -0,0 +1,45 @@
+public function main() returns error? {
+
+    BinaryenModule module = new;
+
+    BinaryenType[] localTypes = ["i32"];
+
+    module.addFunctionImport("println", "console", "log", "i32", "None");
+
+
+    // Get the arguement b
+    BinaryenExpression b = module.localGet(0, "i32");
+
+    // Then call (b == true)
+    BinaryenExpression[] thencallOperands = [makeConst(makeLiteralInt32(1))];
+    BinaryenExpression thenCall = makeCall("println", thencallOperands, 1, "None");
+
+    // Else call (b == false)
+    BinaryenExpression[] elsecallOperands = [makeConst(makeLiteralInt32(0))];
+    BinaryenExpression elseCall = makeCall("println", elsecallOperands, 1, "None");
+
+  
+    Relooper relooper = new(module);
+    RelooperBlock bb0 = relooper.addBlock(makeNop());
```

Comment:
`Relooper relooper = new(module);` is Ok, but let remove `Relooper` from `RelooperBlock`.

Reply:
Isn't relooper.addBlock() similar to callling a function inside the relooper class.

Reply:
Not sure I got the point, but to reiterate I meant to rename `RelooperBlock` to `Block`, `relooper` variable to remain.